src/inputState.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+enum InputState {
+	NONE = 0,
+	PRESSED = 1,
+	HELD = 2,
+	RELEASED = 3
```

Comment:
Do we need a `HELD` state? Can we instead set the state to `HELD` when the key is first pressed, and keep it as `HELD` until the key is released? This way we wouldn't require any `pressedOrHeld` functions.

Reply:
Right now we don't for the game but logically we do want to track if a button is being held or only pressed. Sometimes we want to start an action at a press and do something else during held. This is also better than having a flag in an object updating state.